polaris-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -10,14 +10,21 @@ import {Text} from '../Text';
 import styles from './Breadcrumbs.scss';
 
 export interface BreadcrumbsProps {
-  /** Collection of breadcrumbs */
-  breadcrumbs: (CallbackAction | LinkAction) | (CallbackAction | LinkAction)[];
+  /** @deprecated Collection of breadcrumbs */
+  breadcrumbs?: (CallbackAction | LinkAction) | (CallbackAction | LinkAction)[];
+  /** Breadcrumb link */
+  breadcrumb?: CallbackAction | LinkAction;
```

Comment:
@itwasmattgregg before this ships can we change the name of `breadcrumb` to `backAction`. I am trying to align across the extensions team. See here:
https://github.com/Shopify/app-kit/discussions/150#discussioncomment-4994861